src/middlewares/trip.middleware.js

```diff
@@ -71,6 +72,13 @@ export const validateEditTrip = async (req, res, next) => {
 		);
 		return ResponseService.send(res);
 	}
+	if (req.trip.id !== number) {
+		ResponseService.setError(
+			403,
```

Comment:
is this the right status code?

Reply:
this if condition is useless I was trying something and  I forgot to delete it, let delete it now